### DISCUSSION TITLE
fail to install manager.apk by packageinstaller or adb

### DISCUSSION THREAD

**Discussion by ABPolaris** [LSPosed/LSPosed#1032] [Q&A] [answered]
MIUI 12.5.4(Android 11) on xiaomi MIX 4, fail to install manager.apk but lsposed  installed successfully.
Packageinstaller on MIUI said "Other Problem"(其他错误). 
Adb command is `adb install -r -d manager.apk` and adb err as here:
```
Performing Streamed Install
adb: failed to install manager.apk: Failure [-124: Failed parse during installPackageLI: Targeting R+ (version 30 and above) requires the resources.arsc of installed APKs to be stored uncompressed and aligned on a 4-byte boundary]
```
What`s more, magisk 23 and riru v26 is working.

**Comment by vvb2060** [comment, marked as answer]
无法复现，请确认下载来源。
最新 CI 构建：https://github.com/LSPosed/LSPosed/suites/3643656206/artifacts/88343326

**Reply by ABPolaris** [reply]
唔我好像知道是什么原因了。用面具修补后的boot.img启动的话就会出现这样的问题，小米原版boot.img不会。至于为什么会这样我说不上来（而且大多数应用的安装不受影响）。
